### PR TITLE
Review PR for setup-iot-hub.md

### PR DESCRIPTION
**Pull request by dominicbetts**
# Guidelines for reviewers
Using this Pull Request (PR), you can:

- Add general comments about the article in this PR
- Add a comment to a specific line in one of the files that make up this article
- Propose edits to the article

You can view the published version of this article at:
https://azure.microsoft.com/documentation/articles/setup-iot-hub/

## Add general comments
To add a general comment, use this page (**Conversation** tab)

## Add comments to a specific line
To add a comment to specific line, go to the **Files changed** tab, hover your mouse pointer over the line in the file and click the blue **+** to add a new line comment.

## Propose edits to the article
To propose edits to this article, go to the **Files changed** tab, find the file you want to edit and click the pencil icon (**Change this file using the online editor**), and make your changes. When you are done you will be prompted to create a PR that contains your proposed changes.

> Note: The PR with your proposed changes will be pushing changes to the dominicbetts/azure-content-pr:edit-setup-iot-hub branch.

### CODE REVIEW

File: doc/setup_iothub.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+
+## Add devices to IoT Hub
+
+You must add details your device to IoT Hub before that device can communicate with the hub. When you add a device to an IoT hub, the hub generates the connection string that the device must use when it establishes the secure connection to the IoT hub.
+
+To add a device to your IoT hub, you can use the [iothub-explorer or the Device Explorer][lnk-manage-iothub] utility in this repository ([azure-iot-sdks](https://github.com/Azure/azure-iot-sdks)). These tools will generate a device specific connection string that you need to copy and paste in the source code of the application running on the device.
+
+
+[iothub-landing]: http://azure.microsoft.com/en-us/documentation/services/iot-hub/
```
Comment (sethmanheim) [inline doc/setup_iothub.md]: Remove en-us from URL.

File: doc/setup_iothub.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+[azure-portal]: https://portal.azure.com
+[manage-iothub-portal]: http://azure.microsoft.com/en-us/documentation/articles/iot-hub-manage-portal/
```
Comment (sethmanheim) [inline doc/setup_iothub.md]: Remove en-us from URL.

File: doc/setup_iothub.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+[lnk-manage-iothub]: manage_iot_hub.md
+[resource-group-portal]: https://github.com/Azure/azure-content/blob/master/articles/azure-portal/resource-group-portal.md
```
Comment (sethmanheim) [inline doc/setup_iothub.md]: Why is this pointing to the public repo? Just FYI...

File: doc/setup_iothub.md (hunk 1 above)
Comment (dominicbetts) [inline doc/setup_iothub.md]: Done

File: doc/setup_iothub.md (hunk 2 above)
Comment (dominicbetts) [inline doc/setup_iothub.md]: Done

File: doc/setup_iothub.md (hunk 3 above)
Comment (dominicbetts) [inline doc/setup_iothub.md]: Done

**Comment by dominicbetts** [comment]
Processed all comments up to here

**Pull request closed without merging**